src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
```

Comment:
Good side of @iilyak idea as it also allows to make previously existed resources [gone](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.11) gracefully. It has a difference from regular 404. Not implemented also may have a point for basement of missed features. But that's all about details.